src/explore-education-statistics-admin/src/pages/release/content/components/BasicReleaseSummary.tsx

```diff
@@ -52,43 +55,50 @@ const BasicReleaseSummary = ({ release }: Props) => {
-            <Tag>{getReleaseStatusLabel(release.status)}</Tag>
+          <div className="dfe-flex dfe-align-items--center dfe-justify-content--space-between">
+            <div className="dfe-flex govuk-!-margin-bottom-3">
+              <Tag>{getReleaseStatusLabel(release.status)}</Tag>
+            </div>
+            {releaseTypeIdsToIcons[release.type.id] && (
+              <div className="dfe-flex">
+                <img
+                  src={releaseTypeIdsToIcons[release.type.id].url}
+                  alt={releaseTypeIdsToIcons[release.type.id].altText}
+                  height="120"
+                  width="120"
+                />
+              </div>
+            )}
+          </div>
 
-            <dl className="dfe-meta-content">
-              <dt className="govuk-caption-m">Publish date: </dt>
-              <dd>
-                {release.publishScheduled && (
-                  <strong>
-                    <FormattedDate>
-                      {parseISO(release.publishScheduled)}
-                    </FormattedDate>
-                  </strong>
-                )}
-              </dd>
-              {isValidPartialDate(release.nextReleaseDate) && (
-                <div>
-                  <dt className="govuk-caption-m">Next update: </dt>
-                  <dd>
-                    <strong>
-                      <time>{formatPartialDate(release.nextReleaseDate)}</time>
-                    </strong>
-                  </dd>
-                </div>
+          <SummaryList>
+            <SummaryListItem term="Publish date">
+              {release.publishScheduled ? (
+                <FormattedDate>
+                  {parseISO(release.publishScheduled)}
+                </FormattedDate>
+              ) : (
+                <p>TBA</p>
               )}
-            </dl>
-          </div>
+            </SummaryListItem>
+            {isValidPartialDate(release.nextReleaseDate) && (
+              <SummaryListItem term="Next update">
+                <time>{formatPartialDate(release.nextReleaseDate)}</time>
+              </SummaryListItem>
+            )}
+            {release.updates && release.updates.length > 0 && (
+              <SummaryListItem term="Last updated">
+                <FormattedDate>{release.updates[0].on}</FormattedDate>
 
-          {releaseTypeIdsToIcons[release.type.id] && (
-            <div className="govuk-grid-column-one-quarter">
-              <img
-                src={releaseTypeIdsToIcons[release.type.id].url}
-                alt={releaseTypeIdsToIcons[release.type.id].altText}
-                height="120"
-                width="120"
-              />
-            </div>
-          )}
+                <ReleaseNotesSection release={release} />
+              </SummaryListItem>
+            )}
+            <SummaryListItem term="Receive updates">
+              <a className="dfe-print-hidden govuk-!-font-weight-bold" href="#">
```

Comment:
Is the href meant to be # here? Should it go anywhere?

Reply:
Because this is on the Admin side, we don't have the subscribe to updates page that is on the Public side: https://dev.explore-education-statistics.service.gov.uk/subscriptions?slug=permanent-and-fixed-period-exclusions-in-england

So I've just made the link go nowhere.

There is a potential bit of extra work that could be done to allow prerelease users to subscribe to updates - when prerelease users look at a release before its published, they do that through the admin service. So its possible they might want to subscribe - but I suspect that piece of work is not worth the time investment.